Purchases/Public/RCPurchasesErrorUtils.m

```diff
@@ -62,6 +62,10 @@
             return @"There was an unknown backend error.";
         case RCReceiptInUseByOtherSubscriberError:
             return @"The receipt is in use by other subscriber.";
+        case RCInvalidAppleSubscriptionKeyError:
+            return @"Apple Subscription Key is invalid or it's not present. In order to provide subscription offers, you must first generate a subscription key, please see https://help.apple.com/app-store-connect/#/dev689c93225 for more info";
+        case RCIneligibleError:
+            return @"The User is ineligible for that action.";
```

Comment:
"The user is ineligible for that action."

Is this meant to be vague?

Reply:
Yes. I talked with @rkotzy about it. We want to create a wide error to indicate that the user is ineligible to do a specific action (like the user is ineligible for a discount)